data_services/deploy/ansible/roles/geonode/tasks/main.yml

```diff
@@ -44,7 +44,7 @@
   copy: src=known_hosts dest={{ssh_dir}}/known_hosts
 
 - name: checkout latest web app code
-  git: repo={{code_repository}} version={{app_version}} dest={{app_code_dir}}/{{app_name}} force=yes
+  git: repo={{code_repository}} version=2.4.x dest={{app_code_dir}}/{{app_name}} force=yes
```

Comment:
2.4.x is more stable (2.4 is the latest official release).  Master could
introduce database schema changes or other updates that are incompatible
with dataqs and/or existing Geonode instances.
On Feb 5, 2016 6:09 PM, "Aashish Chaudhary" <notifications@github.com>
wrote:

> In data_services/deploy/ansible/roles/geonode/tasks/main.yml
> <https://github.com/Kitware/minerva/pull/285#discussion_r52085346>:
>
> > @@ -44,7 +44,7 @@
> >    copy: src=known_hosts dest={{ssh_dir}}/known_hosts
> >
> >  - name: checkout latest web app code
> > -  git: repo={{code_repository}} version={{app_version}} dest={{app_code_dir}}/{{app_name}} force=yes
> > +  git: repo={{code_repository}} version=2.4.x dest={{app_code_dir}}/{{app_name}} force=yes
>
> Why hardcode to 2.4.x?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/minerva/pull/285/files#r52085346>.
>